src/pages/workspaces/workspace/analysis/modals/CloudEnvironmentModal.js

```diff
@@ -378,12 +379,13 @@ export const CloudEnvironmentModal = ({
     ])
   }
 
-  const NEW_JUPYTER_MODE = tools.Jupyter.label
-  const NEW_RSTUDIO_MODE = tools.RStudio.label
-  const NEW_GALAXY_MODE = tools.Galaxy.label
-  const NEW_CROMWELL_MODE = tools.Cromwell.label
-  const NEW_AZURE_MODE = tools.Azure.label
```

Comment:
Do we want to remove these constants? I'm almost certain I read that capital case constants go against coding guidelines.

If we don't want to just use the labels instead, is there another suggestion?

Reply:
it's [here](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide#naming-conventions), if this wiki page is up to date:
```
Use camelCase (not UPPER_CASE) variable names, including for those whose values never change. 
```